### PR TITLE
Add an example (wasm web app)

### PR DESCRIPTION
**Pull request by Mubelotix**

### CODE REVIEW

**Comment by Mubelotix** [comment]
@curquiza I used the same HTML and CSS as [crates.meilisearch.com](crates.meilisearch.com). But I am unable to use one hosted CSS file because it contains relative URL paths for fonts so it doesn't work. I had to copy the content of this file and make a few modifications. So the `pkg/index.html` file of this PR contains a lot of things and it's not ideal.

File: examples/web_app/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Meili crates browser 2000
+
+This example is a clone of [crates.meilisearch.com](https://crates.meilisearch.com), but I rewrote the frontend in Rust.\
```
Comment (curquiza) [inline examples/web_app/README.md:3]:
```suggestion
This example is a clone of [crates.meilisearch.com](https://crates.meilisearch.com), but the front-end is written in Rust! The Rust source files are compiled into WebAssembly and so can be readable by the browsers.

```

File: examples/web_app/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+It's using the same MeiliSearch server.
```
Comment (curquiza) [inline examples/web_app/README.md:4]:
```suggestion
This tutorial is using the same MeiliSearch server as the original project, which means you can follow the next steps without running your own MeiliSearch instance.
```

**Review by curquiza — CHANGES_REQUESTED** [review]
Your tutorial is really great!! I requested small changes but the steps are really clear: I followed them without any troubles! Thanks 🙏

**Review by curquiza — APPROVED** [review]
bors merge

**Comment by curquiza** [comment]
Awesome work @Mubelotix! Thank you again!

**Pull request merged**